### ISSUE TITLE
CR History from Version 3+

### ISSUE THREAD

**Issue by AngelaFaulding** [NHSDigital/DataDictionaryPublication#313]
New ticket raised based on three tickets: #189 , #19 and #191, which have been closed.

This new ticket is for item history taken from Version 3+.
All Change Papers need to be available for uses to search for item history, such as from V3+: https://v3.datadictionary.nhs.uk/change_request_log.asp?shownav=1 and the Related DDCN/DSCN/ISN link.

There is a separate ticket to look at the history when we author in Mauro #312.

**Comment by AngelaFaulding** [comment]
Work has been carried out by James and Likin to investigate how to display the CRs on the website when we no longer publish V3+ and migrate to authoring in Mauro.
James is going to produce a test to demonstrate how this can be done.

**Comment by AngelaFaulding** [comment]
When we have the test from James and content has been agreed, Oxygen pages will need updating and uploading to GitHub.

**Comment by AngelaFaulding** [comment]
@likin1 - what is the Jira ticket number for this?

**Comment by AngelaFaulding** [comment]
Jira ticket created: https://nhsd-jira.digital.nhs.uk/browse/DPNDA-317

Further discussion required. Suggested to discuss on the Oxford Call on 25/1/22.

**Comment by AngelaFaulding** [comment]
Discussed on the Oxford call, Likin is investigating.

**Comment by rajabr58** [comment]
Requires some coding so that Likin can process data and send to James.  Will have to wait till next sprint to commence work.

**Comment by GSChandel** [comment]
Likin - Identified the source of data. Need to write the utility, for James W. to work on.

**Comment by AngelaFaulding** [comment]
9/2/22 - Likin has prepared the xml holding CR history information, based on the August 2019 branch as the source. He will  send the xml to James. This will be OK as a test as it will have to be done fully at migration. Likin thinks this will not be an issue at migration.

**Comment by likin1** [comment]
Update: Shared the xml generated from November 2021 with James.

**Comment by AngelaFaulding** [comment]
15/2/22 - James has had a quick look at the output produced by Likin and it appears to have the information that he expected. James needs to investigate further.

**Comment by rajabr58** [comment]
Still waiting for feedback from James

**Comment by jamesrwelch** [comment]
Yes, still waiting for feedback from me - I'll feed back to this ticket when I've had a chance to look at the output and start building some web pages from the content

**Comment by AngelaFaulding** [comment]
Steve to document what is required and advise System C.

**Comment by AngelaFaulding** [comment]
Meeting arranged for 26/06/24 to discuss.

**Comment by AngelaFaulding** [comment]
19/08/24 - Steve is going to talk to Likin about this.

**Comment by AngelaFaulding** [comment]
Steve is talking to Likin about what is required.

**Comment by likin1** [comment]
@stevebowring I am attaching an email from Naresh on this. Has any update happened on this . Am i missing something?
[FW Change History deliverable.txt](https://github.com/user-attachments/files/17316908/FW.Change.History.deliverable.txt)

**Comment by KatePalmerLilley** [comment]
@likin1 could you please add an update

**Comment by AngelaFaulding** [comment]
Naresh believes that the link will only work once a branch has been merged.
Naresh will try and find any information on this.

**Comment by likin1** [comment]
Hi , This history items have been generated

**Comment by AngelaFaulding** [comment]
@likin1 - how do we test this?

**Comment by AngelaFaulding** [comment]
14/04/25 - Likin is working on this.
